runtime/src/main/scala/com/ing/baker/runtime/javadsl/RuntimeEvent.scala

```diff
@@ -9,7 +9,8 @@ import com.ing.baker.types.Value
 
 import scala.collection.JavaConverters._
 
-case class RuntimeEvent(name: String, providedIngredients: util.Map[String, Value])
+case class RuntimeEvent(name: String,
```

Comment:
(minor) Since this is a java api, it is probably overkilled to make it a `case class`, since none will ever use the extra provided stuff.

Reply:
done